src/sentry/static/sentry/app/views/onboarding/configure/index.jsx

```diff
@@ -100,6 +132,9 @@ const Configure = createReactClass({
 
   render() {
     let {orgId, projectId} = this.props.params;
+    let {organization} = this.context;
+    let isExposed =
```

Comment:
this seems like the wrong place for this logic. instead pass organization to the HookStore and let it determine whether to return a component or null. that way sentry code is clean of experiment logic.